experimental/torch_xla2/docs/understand_jax_jit/jax_jit.py

```diff
@@ -0,0 +1,103 @@
+
```

Comment:
do we want to keep all the comments too?

Reply:
yes